src/Symfony/Component/Translation/Tests/DependencyInjection/TranslationPassTest.php

```diff
@@ -25,26 +25,33 @@ public function testValidCollector()
         $loader = (new Definition())
```

Comment:
@chalasr Could you give this test an extra review?

I tried to make it clear that `translation.loader` is a tag.